lib/ioki/model/platform/ride_option.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Ioki
+  module Model
+    module Platform
+      class RideOption < Base
+        attribute :name, on: [:read, :create], type: :string
+        attribute :value, on: :create, type: :string # could also be a boolean or an integer
```

Comment:
I briefly looked into supporting more that one type and it's a little bit more involved. Just feels weird that we might pass a boolean here and have it sent as string. Can the API handle that?

We'd need to adapt [type_cast_attribute_value](https://github.com/ioki-mobility/ioki-ruby/blob/dffab1bdfe37e136ce9728dfd8075d6ab8ed69bb/lib/ioki/model/base.rb#L124-L167) to work with an array of possible types and then check the incoming value against those types. 

Is that worth it?

Reply:
> I briefly looked into supporting more that one type and it's a little bit more involved. Just feels weird that we might pass a boolean here and have it sent as string. Can the API handle that?

No, I don't think so. The API is quite picky about the right data types 😀 I only tested with strings so far.

I added support for it in b9f9212e75a4051dbd5cd8347f0ac687f31828dd.